src/getExtendedPublicKey.c

```diff
@@ -66,6 +66,34 @@ void initializePath(uint8_t *dataBuffer)
 	VALIDATE_PARAM(data.bip32Path[2] >= HARDENED_BIP32);
 }
 
+static void os_swap_bigint32(unsigned char *bigint)
```

Comment:
I guess we might want to extract this into a general and STL-like reverse(begin, end) call ...